cas-server-support-oauth/src/main/java/org/jasig/cas/support/oauth/CentralOAuthService.java

```diff
@@ -216,4 +222,9 @@ AuthorizationCode grantAuthorizationCode(TokenType type, String clientId, String
      * @throws InvalidScopeException the invalid scope exception
      */
     Map<String, Scope> getScopes(Set<String> scopeSet) throws InvalidScopeException;
+
+    /**
+     * @return the token registry.
+     */
+    TokenRegistry getTokenRegistry();
```

Comment:
Expose the `tokenRegistry` in `CentralOAuthService` to `OAuth20RevokeClientTokensController`.

Reply:
Update: a better interface design should hide the token registry but return a list of available tokens associated with the client.